include/bluetooth/gap.h

```diff
@@ -162,6 +162,20 @@ enum {
 	BT_GAP_CTE_NONE = 0xFF,
 };
 
+
+/** @brief Peripheral sleep clock accuracy (SCA) in ppm (parts per million) */
+enum {
+	BT_GAP_SCA_UNKNOWN = 0,
+	BT_GAP_SCA_251_500 = 0,
```

Comment:
Should unknown be the same as 251_500?

Reply:
251_500 is defined with the value 0; but the core spec also states that if unknown, 0 should be used. For the sake of the GAP API, we could have different values, and then convert them when reaching the HCI layer.